spec/slack/web/api/mixins/conversations_spec.rb

```diff
@@ -31,15 +31,39 @@
     end
 
     it 'translates a channel that starts with a #' do
+      expect(conversations).to receive(:conversations_list).with(limit: 100)
```

Comment:
This is a change in default behavior isn't it? This test doesn't pass without the changes.

Reply:
Ah, yes - the old code relies on the default from within the paginator. `Hash#compact` should suffice to fix this if the defaults are nil; see what you think